tsl/test/shared/expected/chunk_move_copy.out

```diff
@@ -0,0 +1,112 @@
+-- This file and its contents are licensed under the Timescale License.
```

Comment:
Mental note; I think Ruslan added a test file `dist_chunk.sql` so we might want to merge this test into that one later.

Reply:
I see that he has added his tests in `chunk_api.sql.in`

Reply:
Sorry, I meant in his yet-to-be-merged PR.